src/emailing_service.py

```diff
@@ -0,0 +1,28 @@
+from decode_function import BucketReader
+from mail_client import MailService
+from dynamo_client import DynamoClient
+from datetime import datetime
+from abc import ABC, abstractmethod
+
+
+class ISendEmail(ABC):
+    @abstractmethod
+    def send_email(self):
+        pass
+
+
+class EmailingService(ISendEmail):
+    def __init__(self, data=dict, reader=BucketReader(), mailer=MailService(), dynamo=DynamoClient(table="DailyTable")):
+        self.reader = reader
+        self.mailer = mailer
+        self.dynamo = dynamo
+        self.bucket = data["bucket"]
+        self.address = data["address"]
+        self.content = data["content"]
+        self.current_date = datetime.today().strftime('%Y.%m.%d.%H.%M.%S')
+        self.content_str = str
+
+    def send_email(self):

```

Comment:
`send_email` should receive the bucket, address and content as separate parameters.